app/models/concerns/reporting/project_data_quality_reports/version_four/support.rb

```diff
@@ -374,18 +361,18 @@ def completeness_support_columns column
         @completeness_support_columns['Exit Response'] = :income_at_later_date_response
       else
         @completeness_support_columns[completeness_metrics[column][:label]] = column
-      # when :veteran
-      #   @completeness_support_columns['Veteran Status'] = :veteran_status
-      # when :ethnicity
-      #   @completeness_support_columns['Ethnicity'] = :ethnicity
-      # when :race
-      #   @completeness_support_columns['Ethnicity'] = :race
-      # when :disabling_condition
-      #   @completeness_support_columns['Disabling Condition'] = :disabling_condition
-      # when :disabling_condition
-      #   @completeness_support_columns['Disabling Condition'] = :disabling_condition
+        # when :veteran
```

Comment:
These are commented out `when` clauses -- move them above the `else` so uncommenting them would be correct syntax?

Reply:
I'm just going to remove them.